bottlerocket/testsys/src/error.rs

```diff
@@ -109,6 +109,9 @@ pub(crate) enum Error {
     #[snafu(display("Error patching {}: {}", what, source))]
     Patch { what: String, source: kube::Error },
 
+    #[snafu(display("Error parsing TOML file: {}", path))]
```

Comment:
Throughout our projects we format errors like this:
- When adding some input information we surround it with single quotes
- When including an underlying error we put it at the end after a colon, without single quotes.

So your error should be

```suggestion
    #[snafu(display("Error parsing TOML file '{}'", path))]
```
Ideally you would have an underlying error for context and it would look like this:

```
    #[snafu(display("Error parsing TOML file '{}': {}", path, source))]
```